report/2.Analysis/Requirements.tex

```diff
@@ -0,0 +1,39 @@
+\section{Requirements}
+In order to be able to conclude upon the success of the project, a fixed set of requirements must be formulated.
+These requirements will serve as a guideline in the conclusion of the report and as markers throughout the project. 
+To formally specify the requirements, the MoSCoW analysis model will be utilized. 
+The MoSCoW model specifies which requirements \textbf{M}ust, \textbf{S}hould, \textbf{C}ould, and \textbf{W}on't be fulfilled to deem the project successful.
+
+\textbf{Must have}
+\begin{itemize}
+\item ability to recognize a moving target in front of the device using an attached camera
+\item ability to hit a target moving with a constant speed along a horizontal axis
+\item move fast enough to keep the laser on the moving target in real time
+\item ability to move the laser at least 60 degrees in both horizontal and vertical directions from the origin.
```

Comment:
denne er rigtig god og præcis. selvom de andre er svære at lave reasonable :P måske skal vi bare have et issue der hedder at rette dem til senere? men hele pointen er jo sådan set at vi laver, hvad vi estimere er rimelige mål, og så kæmper vi for at opfylde dem. måske kan vi ikke... og så har vi noget der næsten kan. og så er det også fint.

Reply:
evt bare de 3 første bliver gået mere i dybden med. og lavet til hvad jeg her kalder testcases og eller scenarier